### PR TITLE
feat: added landing page

### PR DESCRIPTION
**Pull request by AtulHande03a**
Added landing page.
Responsive with desktop as well as with mobile
Screenshot (34)
Screenshot (33)
.

### CODE REVIEW

**Review by shashikantmaurya0007 — APPROVED** [review]
The UI looks elegant and clean .the code is well structured. i like how you have used variables for common css.it looks good to merge:)

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5501
+}
```
Comment (t007rushi) [inline .vscode/settings.json:3]: Can ignore this file in .gitignore

File: index.html [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="author" content="Atul Hande" />
+    <meta name="description" content="Gear UI landing page." />
+
+    <link rel="stylesheet" href="styles.css" />
+    <title>Gear UI</title>
+    <!-- <link rel="shortcut icon" href="favicon.png" type="image/x-icon" /> -->
```
Comment (t007rushi) [inline index.html]: Remove unwanted comments

**Review by t007rushi — COMMENTED** [review]
Naming convention clear and good

File: index.html (hunk 2 above)
Comment (AtulHande03a) [reply index.html]: removed the comments

File: .vscode/settings.json (hunk 1 above)
Comment (AtulHande03a) [reply .vscode/settings.json:3]: Yes, I will have added .gitignore file

**Review by horsemaker — COMMENTED** [review]
Looks good Atul! 👌

**Pull request merged**